ironfish/src/evm/evm.ts

```diff
@@ -33,17 +35,41 @@ export class IronfishEvm {
     const result = await this.runTx(opts)
 
     // TODO(jwp) from custom opcodes populate shields and unshields
+
+    const shields = []
+
+    for (const log of result.receipt.logs) {
+      if (Buffer.from(log[0]).toString('hex') !== 'globalContractAddress') {
```

Comment:
is `'globalContractAddress'` a placeholder here until we determine the hex address?

Reply:
yeah